SnipperAPI/Controllers/SnippetsControllers.cs

```diff
@@ -57,12 +99,35 @@ public ActionResult<IEnumerable<Snippet>> GetAllSnippets()
             }
 
             return Ok(Snippets);
+        }*/
+
+
+        [HttpGet("snippets")]
+        //[Authorize] // Requires authentication
+        public ActionResult<IEnumerable<EncryptedSnippet>> GetAllSnippets()
+        {
+            // Get the current user
+
+            //  list of Snippet objects in your data store
+            if (Snippets.Count == 0)
+            {
+                return NotFound("No snippets found.");
+            }
```

Comment:
Just more optional clean up in C# when you have one line if blocks your actually able to shorten them as you can see in the example below you don't require the block. Optional though just thought I would share.

```suggestion
            //  No Encrypted Snippets in MemoryStore
            if (Snippets.Count == 0)
                return NotFound("No snippets found.");
```